scripts/qesap/test/e2e/test.sh

```diff
@@ -288,3 +288,24 @@ ansible_logs_number=$(find . -type f -name "ansible.*.log.txt" | wc -l)
 # 3 playbooks plus a log file for the initial ansible (not ansible-playbook) call 
 [[ $ansible_logs_number -eq 4 ]] || test_die "ansible .log.txt are not 4 files but has ${ansible_logs_number}"
 rm ansible.*.log.txt
+
+test_step "[test_4.yaml] Run Ansible with --junit"
+find . -type f -name "sambuconero*.xml" -delete || echo "Nothing to delete"
+qesap.py --verbose -b ${QESAPROOT} -c test_4.yaml ansible --junit . || test_die "test_4.yaml fail on ansible"
+junit_logs_number=$(find . -type f -name "sambuconero*.xml" | wc -l)
+[[ $junit_logs_number -eq 1 ]] || test_die "ansible JUNIT reports should be 1 files but are ${junit_logs_number}"
+find . -type f -name "sambuconero*.xml" -delete
+rm ansible.*.log.txt
+
+test_step "[test_4.yaml] Run Ansible with --profile"
+qesap.py --verbose -b ${QESAPROOT} -c test_4.yaml ansible --profile || test_die "test_4.yaml fail on ansible"
+find . -type f -name "sambuconero*.xml" -delete
+rm ansible.*.log.txt
+
+test_step "[test_4.yaml] Run Ansible with --profile and --junit"
```

Comment:
the same for this last test, junit part is ok but https://github.com/SUSE/qe-sap-deployment/pull/254 has improvement for the profile one

Reply:
For example I think that this implementation of the e2e test is not able to spot the issue that this PR is fixing, this test does not fails without your fix in the code and it should.

Reply:
I will change the code later, thanks for the comments.

Reply:
Done. Thanks for the comments.